toolchain/parse/node_kind.cpp

```diff
@@ -47,6 +47,7 @@ auto NodeKind::child_count() const -> int32_t {
   return child_count;
 }
 
+// NOLINTNEXTLINE(readability-function-size): It's hard to extract macros.
 void CheckNodeMatchesLexerToken(NodeKind node_kind, Lex::TokenKind token_kind,
                                 bool has_error) {
```

Comment:
```suggestion
auto CheckNodeMatchesLexerToken(NodeKind node_kind, Lex::TokenKind token_kind,
                                bool has_error) -> void {
```
(Since we're here. Not sure why lint didn't catch this.)

Reply:
clang-tidy does `auto` when there's a return type, but explicitly doesn't hit `void`. It's sad there's not even an option.